client/src/Try/Common.purs

```diff
@@ -0,0 +1,61 @@
+module Try.Common where
+
+import Prelude
+
+{-
+Common values and newtype wrappers
+-}
+--
+-- Page to launch on startup and when clicking home button
+homeRoute :: String
+homeRoute = "/?github=milesfrain/tps/demo/examples/Home.purs"
```

Comment:
Considering changing these paths to expect the `blob` part so they're easier for users to copy while browsing a repo. Then it just needs to be stripped out from the `raw` url for loading.
```
https://github.com               /milesfrain/tps/blob/demo/examples/Home.purs
https://raw.githubusercontent.com/milesfrain/tps/     demo/examples/Home.purs
```